tests/integration/base.py

```diff
@@ -77,6 +77,8 @@ def __init__(self):
 
 
 class TestArgs:
+    __test__ = False
```

Comment:
Why does this line fix the tests?

Reply:
pytest test collection gathers up all classes that start with 'Test', so the __test__ = False marks this class as "not a test".